client/sites/components/sites-dataviews/index.tsx

```diff
@@ -119,16 +122,32 @@ const DotcomSitesDataViews = ( {
 	const fields = useMemo< Field< SiteExcerptData >[] >(
 		() => [
 			{
-				id: 'site',
+				id: 'title-text',
 				label: __( 'Site' ),
-				header: <span>{ __( 'Site' ) }</span>,
-				getValue: ( { item }: { item: SiteExcerptData } ) => item.URL,
+				header: <span></span>,
+				getValue: ( { item }: { item: SiteExcerptData } ) => item.title,
 				render: ( { item }: { item: SiteExcerptData } ) => {
 					return <SiteField site={ item } openSitePreviewPane={ openSitePreviewPane } />;
 				},
 				enableHiding: false,
 				enableSorting: true,
 			},
+			{
+				id: 'site-url',
+				label: __( 'URL' ),
+				header: <span>{ __( 'URL' ) }</span>,
+				render: ( { item }: { item: SiteExcerptData } ) => <SiteUrl site={ item } />,
+				enableHiding: false,
+				enableSorting: true,
+			},
+			{
+				id: 'admin-url',
+				label: __( 'Admin URL' ),
+				header: <span>{ __( 'Admin URL' ) }</span>,
+				render: ( { item }: { item: SiteExcerptData } ) => <SiteAdminUrl site={ item } />,
+				enableHiding: false,
+				enableSorting: true,
```

Comment:
I believe the Admin URL is no longer necessary since it was extracted into actions in https://github.com/Automattic/wp-calypso/pull/96388. 
Would removing it simplify things, or would it not make much of a difference? 😄

Reply:
Should simplify some things, yes.